patches/react-autosize-textarea+7.1.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/react-autosize-textarea/lib/TextareaAutosize.d.ts b/node_modules/react-autosize-textarea/lib/TextareaAutosize.d.ts
```

Comment:
`react-autosize-textarea` is probably no longer maintained because the GitHub repo is archived:
https://github.com/buildo/react-autosize-textarea

We should consider forking it if we need to apply fixes, as it might be easier to change the source file:

https://github.com/buildo/react-autosize-textarea/blob/master/src/TextareaAutosize.tsx

Reply:
This patch is actually pretty simple although it doesn't look like it. It removes two keys that were removed from the React types that were causing issues.

I agree, forking it is a better long term solution so we actually have a maintained version.

Reply:
The patch is simple, but applying further changes might be surprising because of using the patch.

Reply:
There are other issues with this lib. I found the related issue: https://github.com/WordPress/gutenberg/issues/48009. It's also where I learned about the repository being archived. A related comment from @t-hamano about exploring the possibility to replace the library: https://github.com/WordPress/gutenberg/issues/48009#issuecomment-1735720752.

Reply:
In the past, we have attempted to replace this package with another package. However, it was not possible to completely solve the problem. The PR fixing the bug that exists on the library side has also not been merged yet, so that attempt has been closed.

I was also hoping that [CSS `field-sizing`](https://developer.chrome.com/docs/css-ui/css-field-sizing) might be able to completely replace this library, but it hasn't been implemented in all major browsers yet.

We might be better off simulating the behavior of this library or forking this library, as mentioned in [this comment](https://github.com/WordPress/gutenberg/pull/48215#issuecomment-1827645994).

Reply:
I plan to pull this patch out to its own PR as a quick fix to an immediate problem. Other than that I'm not very familiar with this library or what it's used for.

I've created an issue to track work on a better solution: https://github.com/WordPress/gutenberg/issues/61568

Reply:
Extracting here: https://github.com/WordPress/gutenberg/pull/61570